registry-login/src/main.ts

```diff
@@ -0,0 +1,17 @@
+import * as actionsToolkit from '@docker/actions-toolkit'
+
+import * as context from './context'
+import * as docker from './docker'
+
+const inputs: context.Inputs = context.getInputs()
```

Comment:
不用在这里缓存，下面直接用就好了，完整代码见下面的评论
```suggestion
const inputs: context.Inputs = context.getInputs()
```